requirements.txt

```diff
@@ -1,5 +1,7 @@
+terminaltables
```

Comment:
`terminaltables` is an old project and is not under active maintenance. I can't find the usage in the code base. Is there any alternative?

Reply:
Looks like a remnant from a merge conflict since this branch was created before we migrated from `terminaltables` -> `rich`.